helix-view/src/input.rs

```diff
@@ -234,6 +234,26 @@ impl From<crossterm::event::KeyEvent> for KeyEvent {
     }
 }
 
+#[cfg(feature = "term")]
+impl From<KeyEvent> for crossterm::event::KeyEvent {
+    fn from(KeyEvent { code, modifiers }: KeyEvent) -> Self {
+        if code == KeyCode::Tab && modifiers.contains(KeyModifiers::SHIFT) {
+            // special case for Shift-Tab -> BackTab
+            let mut modifiers = modifiers;
+            modifiers.remove(KeyModifiers::SHIFT);
+            crossterm::event::KeyEvent {
+                code: crossterm::event::KeyCode::BackTab,
+                modifiers: modifiers.into(),
+            }
```

Comment:
Why do we need the special case?

Reply:
Because we normalize crossterm's backtab into shift-tab

Reply:
This way it's reversible